{"name": "test_two", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = hasattr('test', 'check')", "trace": "self = <testcase.test_simpe.TestClass object at 0x0000018D9E5C3B00>\n\n    @allure.story(\"测试用例 2\")\n    def test_two(self):\n        x = \"test\"\n>       assert hasattr(x, 'check')\nE       AssertionError: assert False\nE        +  where False = hasattr('test', 'check')\n\ntestcase\\test_simpe.py:15: AssertionError"}, "start": 1597388285711, "stop": 1597388285711, "uuid": "7d8c244c-2cc0-43c3-8af9-da024190d710", "historyId": "4c4a4f015c4cd67ec4d0a3ca0df7b327", "testCaseId": "e763e3ce97be424733a1823e30a71fea", "fullName": "testcase.test_simpe.TestClass#test_two", "labels": [{"name": "story", "value": "测试用例 2"}, {"name": "feature", "value": "测试Dome"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_simpe"}, {"name": "subSuite", "value": "TestClass"}, {"name": "host", "value": "SC-201907251023"}, {"name": "thread", "value": "12248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_simpe"}]}